third_party/amd/lib/TritonAMDGPUToLLVM/BuiltinFuncToLLVM.cpp

```diff
@@ -15,17 +15,19 @@ namespace triton {
 
 using namespace mlir;
 
+using mlir::triton::gpu::appendOrGetExternFuncOp;
```

Comment:
These are not used?

Reply:
Yes these were used for debugging. Removed.